app/models/person.rb

```diff
@@ -56,7 +56,17 @@ def biography
     details["body"]
   end
 
-private
+  def translations
+    available_translations.map do |t|
+      t[:locale] = t[:locale]
+      t[:base_path] = t[:base_path]
+      t[:text] = language_name(t[:locale])
+      t[:active] = true if locale == t[:locale]
+      t
+    end
```

Comment:
I think it's better to be explicit with the hash here, something like this:

```ruby
available_translations.map do |translation|
  translation
    .slice(:locale, :base_path)
    .merge(
      text: language_name(translation[:locale]),
      active: locale == translation[:locale],
    )
end
```

The lines `t[:locale] = t[:locale]`, and `t[:base_path] = t[:base_path]` look a bit odd! :smile:

Reply:
I like that